### ISSUE TITLE
Useless scrollbar on ipad

### ISSUE THREAD

**Issue by hakimio** [DevExpress/devextreme-angular-template#60]
To reproduce:
* Choose device emulation from Chrome developer tools
* Select ipad

You'll see that there is useless scrollbar in "home" page.

**Comment by ovchinnikov** [comment]
The min-height property of dxScrollable content is set to 101%. That is why you see a scrollbar in the iOS simulator. It's necessary to prevent whole page bounce behavior on iPad and iPhone.

**Comment by hakimio** [comment]
@ovchinnikov Why doesn't it happen when "inner toolbar" layout is used then?

**Comment by ovchinnikov** [comment]
I have the same behavior on my side with the "inner toolbar" layout
![inner](https://user-images.githubusercontent.com/1101267/47351151-1473d300-d6c0-11e8-94fe-19a65ad23997.png)

**Comment by hakimio** [comment]
> The min-height property of dxScrollable content is set to 101%. That is why you see a scrollbar in the iOS simulator. It's necessary to prevent whole page bounce behavior on iPad and iPhone.

A simpler way to prevent the whole page bounce would be to set `overflow: hidden` on the html element:
```css
html {
    overflow: hidden;
}
```

**Comment by ovchinnikov** [comment]
Your solution doesn't work for me. Please check this example on a real iOS device: https://qly5o168pq.codesandbox.io/

**Comment by hakimio** [comment]
@ovchinnikov You are right. Unfortunately `overflow: hidden` solution disables page bounce on Mac OS Safari but not on iOS Safari.
